frontend/tests/common/test_message.json

```diff
@@ -41,7 +41,7 @@
       {
         "node_bindings": {
           "n1": [{ "id": "1" }],
-          "n2": [{ "id": "2" }]
+          "n0": [{ "id": "2" }]
```

Comment:
I know that this is just semantic and doesn't really affect anything, but wouldn't it make more sense if `n0` came before `n1` with the `id` of `n0` as 1 and the `id` of `n1` as 2? Just a thought.

Reply:
Updated in 6354b586c0bbfa1213a8289bbdcce81e1e1816d9